sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestUtils.scala

```diff
@@ -464,3 +470,24 @@ private[sql] object SQLTestUtils {
     }
   }
 }
+
+class TestUtilsSuite extends SQLTestUtils with SharedSQLContext {
```

Comment:
Ur, I think we don't need this test suite~

Reply:
Thanks. I will remove it.

Reply:
Thanks!